docs/tr/docs/tutorial/path-params.md

```diff
@@ -0,0 +1,254 @@
+# Yol Parametreleri
+
+Yol "parametrelerini" veya "değişkenlerini" Python string biçimlemede kullanılan aynı söz dizimi ile tanımlayabilirsin.
+
+```Python hl_lines="6-7"
+{!../../../docs_src/path_params/tutorial001.py!}
+```
+
+Yol parametresi olan `item_id`'nin değeri, fonksiyonuna `item_id` argümanı olarak geçirilecektir.
+
+Demek oluyor ki, eğer bu örneği çalıştırırsan ve <a href="http://127.0.0.1:8000/items/foo" class="external-link" target="_blank">http://127.0.0.1:8000/items/foo</a> sayfasına gidersen, şöyle bir yanıt ile karşılaşacaksın:
+
+```JSON
+{"item_id":"foo"}
+```
+
+## Tip içeren yol parametreleri
+
+Standart Python tip anotasyonları kullanarak yol parametresinin tipini fonksiyonda tanımlayabilirsin.
+
+```Python hl_lines="7"
+{!../../../docs_src/path_params/tutorial002.py!}
+```
+
+Bu durumda, `item_id` bir `int` olarak tanımlanacaktır.
+
+!!! check
```

Comment:
`!!! check` admonition'ı için daha [MkDocs - FastAPI Türkiye](https://hasansezertasan.github.io/fastapi-turkiye/tutorials/mkdocs/#admonitions) sayfasında bir sabit belirlememişiz. Doğru çevirisi konusunda emin olamadım.

Yardımcı olabilir misin @alperiox?

Reply:
Şimdilik "Ek bilgi" olarak çevirilmiştir.

Reply:
Tamamdır, gayet güzel. @alperiox, sen ne düşünüyorsun bu konuda?